app/src/main/java/com/keylesspalace/tusky/fragment/SFragment.java

```diff
@@ -146,6 +138,12 @@ protected void more(@NonNull final Status status, View view, final int position)
         final String accountUsername = status.getActionableStatus().getAccount().getUsername();
         final Spanned content = status.getActionableStatus().getContent();
         final String statusUrl = status.getActionableStatus().getUrl();
+        String loggedInAccountId = null;
```

Comment:
Huh? Why it's reset here? Confusing

Reply:
its not reset its initialized

Reply:
Ah, my bad, it seemed like it's the same initialization in another method.